### PR TITLE
TASK: Tweak dev dependencies

### PR DESCRIPTION
**Pull request by kdambekalns**
See https://github.com/neos/flow-development-collection/pull/1584

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,8 +22,7 @@
         "symfony/console": "~2.8",
         "neos/composer-plugin": "^2.0.0",
         "typo3fluid/fluid": "~2.1.3",
-        "ext-mbstring": "*",
-        "ext-xml": "*"
```
Comment (albe) [inline composer.json]: is the ext-xml removed intentionally? That was added with #1357

File: composer.json [hunk 2]
```diff
@@ -103,7 +102,9 @@
     },
     "require-dev": {
         "mikey179/vfsstream": "~1.6",
-        "phpunit/phpunit": "~6.0"
+        "phpunit/phpunit": "~6.0",
+        "doctrine/orm": "~2.5.0",
```
Comment (albe) [inline composer.json]: Readding doctrine as dev dependencies? See #1584

Comment (kdambekalns) [reply composer.json]: Exactly, see #1584 :)

File: composer.json (hunk 1 above)
Comment (kdambekalns) [reply composer.json]: Ok, then it must be added to `Neos.Flow/composer.json` as well!

**Pull request merged**